src/lib/pages/frontsite/Home/Sections/SectionInstallation.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react';
```

Comment:
nit: there is a command in VSCode, `Organize Imports` we have used it in most of our files till now, and we can use it in this one too for the shake of consistency.